jscripts/normalizeCharacters.js

```diff
@@ -0,0 +1,21 @@
+function normalizeCharacters()
+	{
+		// Find searcArg and replace smart quote with single quote 
+		var search = document.getElementById("searchArg").value;
+		search = search.replace("\u2019", "'");
+		document.getElementById("searchArg").value = search;
+		
+		//Advanced search box has three searchArgs
+		var search1 = document.getElementById("searchArg1").value;
```

Comment:
@aprigge I'm not able to get this to work on the Advanced Search screen.  I've tried in Chrome and Firefox, same result: "Search resulted in no hits".

I'm testing with the term from the Jira ticket, `l’Afrique`.  This *does* work on the Basic Search, and on the "New Search" box on search results pages, giving me 3531 results found.

Firefox dev tools shows this after a failure on Advanced Search:
```
TypeError: document.getElementById(...) is null        normalizeCharacters.js:4:25
    normalizeCharacters https://cattest.library.ucla.edu/vwebv/ui/ucladb/jscripts/normalizeCharacters.js:4
    onsubmit https://cattest.library.ucla.edu/vwebv/searchAdvanced:1
```
I think @DRickard is right: You should add some code to confirm that the element you want is present.

Chrome dev tools also shows this error - after I changed console settings to "Preserve log" and unchecked "Group similar".  Looks like there's a similar error in `grayer.js` which is masking this, since it's called first.  I'll make a new ticket for that.